metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -409,6 +412,24 @@ void main() {
       },
     );
 
+    test(
+      ".push() uses the given metrics page factory with correct parameters to create a metrics page",
```

Comment:
🏷️ 
```suggestion
      ".push() uses the given metrics page factory to create a metrics page",
```
Please, apply this suggestion to all similar situations.